File: 100-Days-Series/Day 88/PerfectCubeNumber.js

// Function to find Perfect Cube Number between 1 to N.
const getFindPerfectCubeNumber=(n)=>{
    for(let i=1;i<=n;i++){
        let res=i*i*i;
        if(res>n){
            break;
        }
        console.log(res);
    }
}
// Let's Start Here
let n=30;
console.log("Perfect Cube Number between 1 to "+n+" : ");
getFindPerfectCubeNumber(n);

File: 100-Days-Series/Day 88/PerfectCubeNumber.java
class PerfectCubeNumber {
    // Method to find Perfect Cube Number between 1 to N
    public static void getFindPerfectCubeNumber(int n){
        for(int i=1;i<=n;i++){
            int res=i*i*i;
            if(res>n){
                break;
            }
            System.out.print(res+" ");
        }
    }
    // Main Method
    public static void main(String [] args){
        // Let's Start Here
        int n;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter the value of N : ");
        n=sc.nextInt();
        System.out.println("Perfect Cube Number between 1 to N : ");
        getFindPerfectCubeNumber(n);
    }
}
